superset-frontend/src/explore/components/controls/DndColumnSelectControl/DndFilterSelect.tsx

```diff
@@ -382,7 +382,18 @@ const DndFilterSelect = (props: DndFilterSelectProps) => {
     return new AdhocFilter(config);
   }, [droppedItem]);
 
-  const canDrop = useCallback(() => true, []);
+  const canDrop = useCallback(
```

Comment:
non blocking: does any testable behavior changed with this block? If so, could add include it in the  `DndMetricSelect.test.tsx` file?

Reply:
No. The drag and drop behavior tested in `DndMetricSelect.test.tsx` is related to reordering the metric values. Testing drag and drop from the datasource panel to the control field is more complicated - I suggest we add those tests in a separate PR, both for filters and metrics